test/integration/testManualConnection.js

```diff
@@ -231,8 +231,36 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
       });
     });
 
+    describe('Connection - MFA authenticator with DUO', function () {
+      const connectionOption = connOption.MFA;
+    
+      it('test - authentication with passcode', function (done) {
+        const connection = snowflake.createConnection(connectionOption);
+        connection.connectAsync(function (err) {
+          try {
+            assert.ok(!err);
+            done();
+          } catch (err){
+            done(err);
+          }
+        });
+      });
+  
+      it.skip('test - authentication with passcodeInPassword', function (done) {
+        const connection = snowflake.createConnection({ ...connectionOption, passcodeInPassword: true });
+        connection.connectAsync(function (err) {
+          try {
+            assert.ok(!err);
+            done();
+          } catch (err){
+            done(err);
+          }
+        });
+      });
+    });
+
     describe('Connection - MFA authenticator', function () {
-      const connectionOption = { ...connOption.valid, authenticator: 'USERNAME_PASSWORD_MFA', clientRequestMFAToken: true };
+      const connectionOption = { ...connOption.MFA, passcode: null, clientRequestMFAToken: true };
```

Comment:
connOption.MFA doesn't set expected authenticator -> authenticator: 'USERNAME_PASSWORD_MFA'

Reply:
I added it.